project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.16
```

Comment:
It is good to set the editor to add an end of line before the end of file automatically.